### ISSUE TITLE
Captum with recurrent units (LSTMs)

### ISSUE THREAD

**Issue by fahadahaf** [pytorch/captum#328]
Hi, 

My model uses a bi-directional LSTM and using an attribution method while the model is in evaluate mode throws the following error:
**cudnn RNN backward can only be called in training mode**

I tried DeepLiftSHAP attribution method without the model being in eval mode and it works (gives me the right attributes) but I am afraid thats not the right way? This will mutate the weights?

Kindly suggest how to tackle the problem (also when I am using other methods such as DeepLIFT and Integrated Gradients).

Thank you

**Comment by NarineK** [comment]
Thank you for the question, @fahadahaf !

Try to set ` torch.backends.cudnn.enabled=False` and see if that helps.

**Comment by fahadahaf** [comment]
Awesome! thank you @NarineK, that worked.
